From eabcf6f2d1ca4685576fb0b5ee67b13e39ef7abf Mon Sep 17 00:00:00 2001
From: Laurent FERT <laurent.fert@intel.com>
Date: Tue, 6 Aug 2013 17:49:13 +0200
Subject: [PORT FROM MAIN] grub: add LOCAL_ASFLAGS

MR2 has a patch in the build system disabling PRIVATE_CFLAGS for asm
compilation.

Add LOCAL_ASFLAGS to all modules with the same content as LOCAL_CFLAGS.

Issue: BZ: 128865
Change-Id: I11ae7efb4258f76374bc4040ed87a8d2690ea1b3
Signed-off-by: Laurent FERT <laurent.fert@intel.com>
---
 Android.mk |    6 ++++++
 1 files changed, 6 insertions(+), 0 deletions(-)

diff --git a/Android.mk b/Android.mk
index ae94276..44016d6 100644
--- a/Android.mk
+++ b/Android.mk
@@ -32,6 +32,8 @@ LOCAL_CFLAGS := \
         -falign-functions=1 -Wundef
 LOCAL_CFLAGS += -m32 -O2 -fno-builtin -nostdinc -fno-reorder-functions -fno-stack-protector
 
+LOCAL_ASFLAGS := $(LOCAL_CFLAGS)
+
 LOCAL_C_INCLUDES := $(LOCAL_PATH)/stage1
 
 LOCAL_MODULE := grub_stage1
@@ -93,6 +95,8 @@ LOCAL_CFLAGS += -DHAVE_CONFIG_H -DFSYS_EXT2FS=1 -DSUPPORT_SERIAL=1
 
 LOCAL_CFLAGS += -DPRESET_MENU_EXTERNAL
 
+LOCAL_ASFLAGS := $(LOCAL_CFLAGS)
+
 LOCAL_C_INCLUDES := \
 	$(LOCAL_PATH)/stage1 \
 	$(LOCAL_PATH)/stage2
@@ -128,6 +132,8 @@ LOCAL_CFLAGS += -m32 -Os -fno-builtin -nostdinc -fno-reorder-functions -fno-stac
 
 LOCAL_CFLAGS += -DHAVE_CONFIG_H -DFSYS_EXT2FS=1 -DSUPPORT_SERIAL=1
 
+LOCAL_ASFLAGS := $(LOCAL_CFLAGS)
+
 LOCAL_C_INCLUDES := \
 	$(LOCAL_PATH)/stage1 \
 	$(LOCAL_PATH)/stage2
-- 
1.7.4.1

